README.md

```diff
@@ -171,7 +171,7 @@ To enable this image reporter, add it to your `jest.config.js` "reporters" defin
 
 #### Usage in TypeScript
 
-In TypeScript, you can declare `toMatchImageSnapshot` like this:
+In TypeScript, you can use the [DefinitelyTyped](https://www.npmjs.com/package/@types/jest-image-snapshot) definition or declare `toMatchImageSnapshot` like this:
```

Comment:
Can you add a note on how that package is not maintained by the jest-image-snapshot maintainers and so may be out of date/not accurate but we don't officially support it?

Reply:
I added that note